browser/brave_content_browser_client.cc

```diff
@@ -1267,6 +1272,21 @@ BraveContentBrowserClient::CreateThrottlesForNavigation(
       throttles.push_back(std::move(ai_chat_throttle));
     }
   }
+
+  if (Profile::FromBrowserContext(context)->IsRegularProfile()) {
+    std::unique_ptr<ai_chat::AIChatBraveSearchThrottle::Delegate> delegate;
+#if !BUILDFLAG(IS_ANDROID)
+    delegate =
+        std::make_unique<ai_chat::AIChatBraveSearchThrottleDelegateImpl>();
+#endif
```

Comment:
so, basically we don't create the throttle on Android? (as we return `nullptr` if `delegate` is `nullptr`)

Could we move the `#if` around this whole block for now?

Reply:
I was thinking about the same thing, but actually I think it would be better to pass a callback `AIChatBraveSearchThrottleDelegate::Create`. That way we only create the delegate if we actually need it (all other checks pass)